src/main/java/brandkon/Product/Product.java

```diff
@@ -0,0 +1,77 @@
+package brandkon.Product;
+
+import brandkon.Brand.Brand;
+import jakarta.persistence.*;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import java.time.LocalDateTime;
+
+@Entity
+public class Product {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String brandName;  //Brand에 name이 있기 때문에 Product에 brandName을 쓸 필요도 없고, 관련도 없다.(중복데이터)
+    private String productName;
```

Comment:
```suggestion
    private String name;
```
`product.getProductName()` 처럼 쓰는 것보다는 `product.getName()`이 나을 거에요